docs/model_control_mode.md

```diff
@@ -0,0 +1,68 @@
+# Model Control Mode
+
+TorchServe now supports model control mode with two settings "none"(default) and "explicit"
+
+## Two ways to set Model Control
+1. Add `--model-mode-explicit` to command line when running TorchServe to switch from none to explicit mode. Command line cannot be used to set mode to none, can only be used to set to explicit
+2. Add `model_control_mode=explicit` or `model_control_mode=none` to config.properties file
+
+Priority between cmd and config file follows the following [TorchServer standard](https://github.com/pytorch/serve/blob/c74a29e8144bc12b84196775076b0e8cf3c5a6fc/docs/configuration.md#advanced-configuration)
+* Example 1:
+  * Config file: `model_control_mode=none`
+
+    cmd line: `torchserve --start --ncs --model-store model_store --model-mode-explicit`
+
+    Result: Explicit mode enabled
+* Example 2:
+  * Config file: `model_control_mode=explicit`
+
+    cmd line: `torchserve --start --ncs --model-store model_store`
+
+    Result: Mode is explicit (no way to enable "none" through cmd)
+
+## Model Control Mode Default
+At startup TorchServe loads only those models specified explicitly with the `--models` command-line option. After startup users will be unable to register or delete models in this mode.
```

Comment:
The naming of the modes (none, explicit) does not make sense for TorchServe. Using the wording "explicitly" in the doc for the default (none) mode will confuse users even more.

The difference between other solutions (where this naming makes sense) is that TorchServe does not load all models in model_store by default but none. Only if we give --models we load the specified. So for TorchServe this PR is more about locking the api after startup so no models can be loaded afterwards. This should also be reflected in the parameter. 

To make clear what the parameter does we will need to name it something like: --model-control-api-enabled

Reply:
changed it to --model-api-enabled and changed explicit to enabled in docs and code